polkadot/runtime/westend/src/lib.rs

```diff
@@ -1163,15 +1164,18 @@ impl pallet_message_queue::Config for Runtime {
 impl parachains_dmp::Config for Runtime {}
 
 parameter_types! {
-	pub const DefaultChannelSizeAndCapacityWithSystem: (u32, u32) = (4096, 4);
+	pub const HrmpChannelSizeAndCapacityWithSystemRatio: Percent = Percent::from_percent(100);
```

Comment:
I thought we wanted half of the max? Or here it's 100 because it's a testnet?

Reply:
Well, judging based on the linked [comment from the description](https://github.com/paritytech/polkadot-sdk/pull/3721/files#r1541001420),  "half of the max value" it was somewhat of a blind guess (no offence).

If you check this (not sure if still actual): https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/roadmap/implementers-guide/src/runtime/hrmp.md
```
 ii. Check that proposed_max_capacity is less or equal to config.hrmp_channel_max_capacity and greater than zero.
iii. Check that proposed_max_message_size is less or equal to config.hrmp_channel_max_message_size and greater than zero.
```
and also the implementation [here](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/parachains/src/hrmp.rs#L1448-L1456), I think, there shouldn't be a problem with setting those values to 100% (e.g. `config.hrmp_channel_max_capacity`).

Also, maybe we don't even need [`type DefaultChannelSizeAndCapacityWithSystem`](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/parachains/src/hrmp.rs#L279) and we could just use a max  from `ActiveConfig` e.g. `config.hrmp_channel_max_capacity`.

Reply:
that was just an arbitrary number I made and it is for testnet so shouldn't really matter much